app/gui2/e2e/undoRedo.spec.ts

```diff
@@ -10,18 +11,18 @@ test('Adding new node', async ({ page }) => {
   await locate.addNewNodeButton(page).click()
   await expect(locate.componentBrowserInput(page)).toBeVisible()
   await page.keyboard.insertText('foo')
-  await page.keyboard.press('Control+Enter')
+  await page.keyboard.press(`${CONTROL_KEY}+Enter`)
```

Comment:
Interesting. Does it mean that E2E tests never worked on mac?

Reply:
This one has never worked, I think it's fairly recent